test/e2e/storage/persistent_volumes-local.go

```diff
@@ -552,7 +593,12 @@ func cleanupLocalVolume(config *localTestConfig, volume *localTestVolume) {
 	if volume.localVolumeType == TmpfsLocalVolumeType {
 		unmountTmpfsLocalVolume(config, volume.hostDir)
 	}
-
+	if volume.localVolumeType == GCELocalSSDVolumeType {
```

Comment:
I would do a similar refactoring for the cleanup

Reply:
i actually tried but realized that cleanup is slightly different because the volume type is gotten from the volume itself. But theres also the possibility of the volume being nil and is checked for at the start of this function. If we wanted to split it into multiple functions through a map like the setup we would have to move all of this logic outside of the function which seems a little odd to me. What do you think?

Reply:
I think it's ok to only call the type-specific methods after checking for nil.